File: src/main/java/ru/darkkeks/vkmirror/vk/object/ChatSettings.java
package ru.darkkeks.vkmirror.vk.object;

import com.google.gson.annotations.SerializedName;

import java.net.URL;

public class ChatSettings {

    @SerializedName("photo")
    private Photos photos;

    @SerializedName("title")
    private String title;

    public Photos getPhotos() {
        return photos;
    }

    public String getTitle() {
        return title;
    }

    public static class Photos {
        @SerializedName("photo_50")
        private URL photo50;

        @SerializedName("photo_100")
        private URL photo100;

        @SerializedName("photo_200")
        private URL photo200;


        public URL getPhoto50() {
            return photo50;
        }

        public URL getPhoto100() {
            return photo100;
        }

        public URL getPhoto200() {
            return photo200;
        }
    }
}

File: src/main/java/ru/darkkeks/vkmirror/vk/object/ChatSettings.kt
package ru.darkkeks.vkmirror.vk.`object`

import com.google.gson.annotations.SerializedName
import java.net.URL

class ChatSettings {
    @SerializedName("photo")
    val photos: Photos? = null

    val title: String? = null

    class Photos {
        @SerializedName("photo_50")
        val photo50: URL? = null

        @SerializedName("photo_100")
        val photo100: URL? = null

        @SerializedName("photo_200")
        val photo200: URL? = null
    }
}